math/geometry/trigan/main_function.go

```diff
@@ -0,0 +1,17 @@
+// Package trigan for trigonometric functions and algorithms
+package trigan
+
+import "math"
+
+// Cos  returns the cosine of the radian argument x. [See more](https://en.wikipedia.org/wiki/Sine_and_cosine)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see sqrt_test.go
+
+func Cos(x float64) float64 {
+	tp := 1. / (2. * math.Pi)
+	x *= tp
+	x -= .25 + math.Floor(x+.25)
+	x *= 16. * (math.Abs(x) - .5)
+	x += .225 * x * (math.Abs(x) - 1.) //Extra precision
+	return x
```

Comment:
Good algorithm. Just a note though - I think you should make your `float`s explicit. Like `1.0` instead of `1.` Everyone  who has coded would know about this but its better to be explicit still. Adding a `0` pre/post a number doesn't hurt.

Second, I think we already have the `math.Abs` and `math.Floor` in the `math` package - kind of a redundant thing if you already have `Cos` in the standard library.

Also, `Cos` is not only relevant to geometry but rather the whole math package - so should live in the math package.

Reply:
Good! Done

Reply:
> Also, Cos is not only relevant to geometry but rather the whole math package - so should live in the math package.

I think you missed this part of the comment...

Reply:
I'm sorry, my commit with edits, for some reason, didn't pass right away and I didn't notice it